CMakeLists.txt

```diff
@@ -0,0 +1,200 @@
+SET(PACKAGE osm2pgsql)
+SET(PACKAGE_NAME osm2pgsql)
+SET(PACKAGE_VERSION 0.89.0-dev)
+
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.7)
+
+PROJECT(osm2pgsql)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+
+SET(DATA_DIR \".\")
+
+OPTION(TEST_BUILD "Build with tests and strict options" OFF)
+
+if (NOT WIN32 AND NOT APPLE)
+  # No need for this path, just a workaround to make cmake check pass on all systems
+  set(PostgreSQL_TYPE_INCLUDE_DIR /usr/include)
+endif()
+
+# Just in case user installed RPMs from http://yum.postgresql.org/
+list(APPEND PostgreSQL_ADDITIONAL_SEARCH_PATHS /usr/pgsql-9.3 /usr/pgsql-9.4)
+
+if ("${PROJECT_SOURCE_DIR}" STREQUAL "${PROJECT_BINARY_DIR}")
+  message(FATAL_ERROR "In-source builds are not allowed, please use a separate build directory like `mkdir build && cd build && cmake ..`")
+endif()
+
+message(STATUS "Building osm2pgsql ${PACKAGE_VERSION}")
+
+if (NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE=Release)
```

Comment:
This sets a variable CMAKE_BUILD_TYPE=Release, you want to replace the = with a space to set CMAKE_BUILD_TYPE.